provisioning/osx/configure/src/args.js

```diff
@@ -2,29 +2,38 @@ export default {
   file: {
     alias: 'f',
     demandOption: true,
+    requiresArg: true,
     default: 'stacks.yml',
     describe: 'YAML file with stacks configuration',
     type: 'string',
   },
+  domain: {
+    alias: 'd',
+    demandOption: true,
+    requiresArg: true,
+    default: 'local',
+    describe: 'The name of the top-level domain you want to use',
+    type: 'string',
+  },
   update: {
     alias: 'u',
     demandOption: true,
     default: false,
     describe: 'If true, updates the load balancer with new ports',
     type: 'boolean',
   },
-  domain: {
-    alias: 'd',
-    demandOption: true,
-    default: 'local',
-    describe: 'The name of the top-level domain you want to use',
+  deploy: {
+    demandOption: false,
+    describe: 'A comma separated list of stacks to deploy',
     type: 'string',
   },
-  manager: {
-    alias: 'm',
+  swarm: {
+    alias: 's',
     demandOption: true,
+    requiresArg: true,
     default: 'mgr1',
-    describe: 'The name of a manager in the swarm (the docker-machine VM)',
+    describe:
+      "The docker-machine name of a manager node in a local swarm, OR the path to Docker's UNIX socket file",
```

Comment:
Might be a good idea to have a `swarms.yml` (or something like that) that has the list of named available deployment targets and the different ways to talk to them (docker machine ssh, socket, normal ssh, direct docker API...), then you can just pass in the name of the swarm. 

On top of that you could have an "add-swarm" command to write into that file :)